test/test_heuristics.rb

```diff
@@ -91,6 +91,13 @@ def test_es_by_heuristics
     })
   end
 
+  def test_eslintrc_by_heuristics
+    assert_heuristics({
+      "JSON" => all_fixtures("JSON", ".eslintrc"),
```

Comment:
> you could simply use

That does not work. The heuristic under question is only testing `.eslintrc` file. The rest of the files under samples/JSON dir (for example `*.lock`) are listed in `languages.yml` as extensions, which are not classified by heuristic at all.

Looking at the rest of this file how single files are handled, I have made the change accordingly:

```
"JSON": ["samples/JSON/filenames/.eslintrc"]
```

rebased and repushed